src/main/java/com/spochi/entity/Initiative.java

```diff
@@ -47,6 +48,12 @@ public InitiativeResponseDTO toDTO() {
         dto.setStatus_id(this.statusId);
         dto.setImage(this.image);
 
+        if (user_id.equals(this.userId)) {
+            dto.setIs_from_current_user(true);
+        } else {
+            dto.setIs_from_current_user(false);
+        }
```

Comment:
Se puede simplificar:
setIs_From_current_user(user_id.equals(this.userId))

En general siempre que se cambia un flag dependiendo de un if se puede simplificar de esa forma

Reply:
ayer cuando lo hice me sonó una alarma, sabía que tenía que haber una forma mas simple de hacerlo, pero estaba quemada y no se me ocurría. gracias por el dato!!